src/main.rs

```diff
@@ -107,28 +82,15 @@ impl Command {
                         Command {
                             name: "exit",
                             help: "To exit the curent session",
-                            main: Box::new(|_: &Vec<String>,
-                                            _: &mut BTreeMap<String, String>,
-                                            _: &mut Vec<Mode>| {}),
+                            main: Box::new(|_: &Vec<String>, _: &mut Shell| {}),
```

Comment:
This exists so that there is a help entry for exit. It should really abort the current process. I don't remember why, but there is a reason it is the way it is right now. I will open an issue to address this.